pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -143,7 +143,7 @@ func TestGetClusterMasterClient(t *testing.T) {
 	c := spltest.NewMockClient()
 	mgr.c = c
 	cm := mgr.getClusterMasterClient()
-	if cm.ManagementURI != "https://splunk--cluster-master-service.test.svc.cluster.local:8089" {
+	if cm.ManagementURI != splcommon.ManagerServiceTest {
```

Comment:
I feel it is more readable if you can just replace the `master` keyword alone with variable substitution.  Same comment with other places.

Reply:
Let's discuss this suggested approach. I opened this mock PR for comparison: https://github.com/splunk/splunk-operator/pull/540/files

Reply:
Resolved in the latest commit which we reduced the amount tokenizations and improved the naming convention.